docs/writing-tests/crashtest.md

```diff
@@ -12,11 +12,10 @@ The simplest crashtest is a single HTML file with any content. The
 test passes if the load event is reached, and the browser finishes
 painting, without terminating.
 
-In some cases crashtests may need to perform work after the initial
-page load. In this case the test may specify a `class=wait` attribute
-on the root element. The test will not complete until that attribute
-is removed from the root. At the time when the test would otherwise
-have ended a `TestRendered` event is emitted; test authors can use
-this event to perform modifications that are guaranteed not to be
-batched with the initial paint. This matches the behaviour of
-[reftests](reftests).
+In some cases crashtests may need to perform work after the initial page load.
+In this case the test may specify a `class=test-wait` attribute on the root
```

Comment:
`class=test-wait` is the real change here, hard to spot because of line wrapping.